go.mod

```diff
@@ -9,58 +9,51 @@ require (
-	k8s.io/component-base v0.18.6
-	k8s.io/klog v1.0.0
 	k8s.io/kubelet v0.0.0
 	k8s.io/kubernetes v1.18.6
-	k8s.io/utils v0.0.0-20200729134348-d5654de09c73
 	sigs.k8s.io/yaml v1.2.0
 )
 
 // The k8s "sub-"packages do not have 'semver' compatible versions. Thus, we
 // need to override with commits (corresponding their kubernetes-* tags)
 replace (
-	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.4.0
-	github.com/swatisehgal/topologyapi => github.com/swatisehgal/topologyapi v0.0.0-20201002094043-bc432ffbe41c
 	//force version of x/text due CVE-2020-14040
 	golang.org/x/text => golang.org/x/text v0.3.3
-	k8s.io/api => k8s.io/api v0.18.6
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.18.6
-	k8s.io/apimachinery => k8s.io/apimachinery v0.18.6
-	k8s.io/apiserver => k8s.io/apiserver v0.18.6
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.18.6
-	k8s.io/client-go => k8s.io/client-go v0.18.6
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.18.6
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.18.6
-	k8s.io/code-generator => k8s.io/code-generator v0.18.6
-	k8s.io/component-base => k8s.io/component-base v0.18.6
-	k8s.io/cri-api => k8s.io/cri-api v0.18.6
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.18.6
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.18.6
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.18.6
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.18.6
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.18.6
-	k8s.io/kubectl => k8s.io/kubectl v0.18.6
-	k8s.io/kubelet => k8s.io/kubelet v0.18.6
-	k8s.io/kubernetes => github.com/swatisehgal/kubernetes v1.19.0-beta.2.0.20200928123437-1c4f5ad40262
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.18.6
-	k8s.io/metrics => k8s.io/metrics v0.18.6
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.18.6
+	google.golang.org/grpc => google.golang.org/grpc v1.27.1
+	k8s.io/api => k8s.io/api v0.19.4
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.19.4
+	k8s.io/apimachinery => k8s.io/apimachinery v0.19.4
+	k8s.io/apiserver => k8s.io/apiserver v0.19.4
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.19.4
+	k8s.io/client-go => k8s.io/client-go v0.19.4
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.19.4
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.19.4
+	k8s.io/code-generator => k8s.io/code-generator v0.19.4
+	k8s.io/component-base => k8s.io/component-base v0.19.4
+	k8s.io/cri-api => k8s.io/cri-api v0.19.4
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.19.4
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.19.4
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.19.4
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.19.4
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.19.4
+	k8s.io/kubectl => k8s.io/kubectl v0.19.4
+	k8s.io/kubelet => k8s.io/kubelet v0.19.4
+	k8s.io/kubernetes => k8s.io/kubernetes v1.19.4
```

Comment:
We would still need to replace kubernetes 1.19.4 by a customized version as the kubernetes 1.19.4 does not have the podresource API changes we need (GetAvailable/GetAllocatable and cpuid and device topology information)  e.g.
`k8s.io/kubernetes => github.com/swatisehgal/kubernetes v1.19.0-beta.2.0.20200928123437-1c4f5ad40262` in order to be able to access the API.

Reply:
true, missed it during the rebase

Reply:
done